cmd/nginx/flags.go

```diff
@@ -121,6 +121,10 @@ different namespace than their own. May be used together with watch-namespace.`)
 			`Update the load-balancer status of Ingress objects when the controller shuts down.
 Requires the update-status parameter.`)
 
+		waitBeforeShutdown = flags.Int("wait-before-shutdown", 0,
+			`Delay shutting down nginx after updating the healthcheck endpoint, to give time for 
```

Comment:
> after updating the healthcheck endpoint

after a termination signal is received

> to give time for external load balancers to update their statuses.

to give time for external load balancers to update their statuses before the shutdown.

Reply:
Thanks! changing